src/site/constants/drupals.js

```diff
@@ -26,8 +26,8 @@ const DRUPAL_PROD = {
  * @module site/constants/drupals
  */
 const DRUPALS = {
-  [ENVIRONMENTS.LOCALHOST]: DRUPAL_STAGING,
-  [ENVIRONMENTS.VAGOVDEV]: DRUPAL_STAGING,
+  [ENVIRONMENTS.LOCALHOST]: DRUPAL_DEV,
```

Comment:
I notice this while working on the changes here. We should probably point to dev, since we normally match up with what dev.va.gov does locally.